lib/utils/request.js

```diff
@@ -535,4 +536,31 @@ function getProxy() {
   var proxy = fhc.config.get("proxy");
   log.silly(proxy, "getProxy");
   return proxy;
+}
+
+/**
+ * This method can do a request as multipart/form-data
+ * key value pairs, files and buffers.
+ * @param url
+ * @param dataObject - { name: name, value:value }
+ * @param cb
+ * @param method
+ */
+function doRequestWithDataObjectValues(url, dataObject, cb, method) {
+
+  var fullUrl = urlUtils.resolve(getFeedHenryUrl(), url);
+
+  var opts = {
+    url : fullUrl,
+    headers : getHeaders(),
+    method: method || 'post',
+    proxy : getProxy()
+  };
+
+  var req = requestjs(opts,uploadHandlerFor(cb));
```

Comment:
Calling it this way you swallow the success and error messages. I suggested some change few comments above, that change to work requires this piece of code instead of that line:

var req = requestjs(opts,function(error, response, body) {

    if (error) {
      cb(error);
    }

    var result = JSON.parse(body);

    if (result.status === 'error') {
      return cb(result.message);
    } else {
      return cb(null, result);
    }

  });

I have to admin this is not very nice but at least user is informed about the success and also more detailed error message is displayed.